Makefile

```diff
@@ -47,8 +47,14 @@ GDB     = $(CROSS_COMPILE)gdb
 NRFUTIL = adafruit-nrfutil
 NRFJPROG = nrfjprog
 
+# Set make directory command, Windows tries to create a directory named "-p" if that flag is there.
+ifneq ($(OS), Windows_NT)
 MK = mkdir -p

```

Comment:
if possible, could you please put 2 spaces indent to the MK in the `ifneq`, it makes thing a bit easier to read. This is totally optional

Reply:
Done&done! I actually had intended to do that, but used a tab, and was met with a "recipe started before target" and forgot to go back and explore why it wouldn't let me do that.

But I promise I won't get on my "white space shouldn't define structure in languages" soapbox today 🙂

Reply:
yeah, tab will trigger makefile to execute recipe rule, space is harmless. It is really frustrating :)

Reply:
Well there you have it, I learn something new every day!